app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "mv" and command[2][-1] != "/":
```

Comment:
Also, I think it's not a great idea to put your whole function into an `if` block. I would suggest to reformat it like this:
```python
if ...:
    return
...

Reply:
And here's graphical representation of the problems this style presents, in my opinion. :)
![image](https://user-images.githubusercontent.com/108597725/194049480-705e370f-1892-41f3-b155-78e9cce0ae09.png)

Reply:
Good point